run-a-node.md

```diff
@@ -163,6 +163,12 @@ kujirad keys add validator
 ```
 copy the seed phrase and put it somewhere safe.
 you will need to also make note of the address "kujira..." and use that in the faucet to get some coins.
+
+If you already have a wallet, restore it using a seed phrase:
+```
+kujirad keys add validator --recover 
```

Comment:
`validator` needs to be in some kind of brace consistent with the rest of the docs to indicate it's a user selected varaible